src/test/kotlin/com/ampnet/reportserviceeth/service/BlockchainServiceTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.ampnet.reportserviceeth.service
+
+import com.ampnet.reportserviceeth.blockchain.BlockchainService
+import com.ampnet.reportserviceeth.blockchain.BlockchainServiceImpl
+import com.ampnet.reportserviceeth.config.ApplicationProperties
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.junit.jupiter.SpringExtension
+
+@SpringBootTest
+@ActiveProfiles("secret")
+@ExtendWith(SpringExtension::class)
+@Disabled("Not for automated testing")
```

Comment:
Secrets are not needed because the connection to the blockchain is disabled for automated testing.